### ISSUE TITLE
does it work with Expo?

### ISSUE THREAD

**Issue by rhyannc** [zaguiini/react-native-pure-jwt#37]
I use expo and I installed it through Yarn, but it didn't work, the error appears,
TypeError: null is not an object (evaluating 'RNPureJwt.sign')

Any solution?
Thanks

**Comment by mentor971** [comment]
same issue for me

**Comment by chompasina** [comment]
Same here. It does not seem to work with Expo.

**Comment by zaguiini** [comment]
No, it needs native bindings, so it does not work with Expo unless Expo supports installing native modules.
